quic/s2n-quic-crypto/src/aes/testing.rs

```diff
@@ -27,7 +27,10 @@ macro_rules! aes_impl {
 
             lazy_static! {
                 static ref IMPLEMENTATIONS: Vec<Implementation> = {
+                    #[cfg(any(target_arch = "x86", target_arch = "x86_64", test))]
                     let mut impls = vec![];
+                    #[cfg(not(any(target_arch = "x86", target_arch = "x86_64", test)))]
```

Comment:
I think it might be cleaner to pass the owned vec and return it in the `implementations` call. So basically:

```rust
let impls = vec![];

#[cfg(...)]
let impls = x86::implementations(impls);
```

Reply:
yeah thats better, thanks